README.md

```diff
@@ -103,13 +103,6 @@ AddCreditSmallMessage(id_=42, days=7, secret_key=secret_key).to_keycode()
 # outputs 135 242 422 455 244
 ```
 
-Update Credit
-
-```python
```

Comment:
This was incorrect - not supported even in previous version.

Reply:
lol

Reply:
It was supported way back in 1.0, but we forgot to remove this example when we removed "update credit".